rocketbot/commands/dms.py

```diff
@@ -0,0 +1,55 @@
+from typing import List, Tuple
+import subprocess
+import os
+import re
+
+import dmsclient as dms
+
+import rocketbot.commands as c
+import rocketbot.master as master
+import rocketbot.models as m
+
+
+class DMS(c.BaseCommand):
+    def __init__(self, master: master.Master, token: str):
+        self._create_dmsclient_config_if_missing(token)
+        self.master = master
+
+    def usage(self) -> List[Tuple[str, str]]:
+        return [
+            ('order', 'Orders product in dms for yourself.'),
+        ]
+
+    def can_handle(self, command: str) -> bool:
+        """Check whether the command is applicable
+        """
+        return command in ['dms', 'drink', 'drinks', 'order']
+
+    async def handle(self, command: str, args: str, message: m.Message) -> None:
+        """Handle the incoming message
+        """
+        argv = re.split(r'\s+', args.strip())
+
+        userargv = [arg for arg in argv if arg.startswith('-u') or arg.startswith('--user')]
+        if len(userargv) == 0:
+            userid = message.u.username
+            argv.append('--user={}'.format(userid))
+        if '--force' not in argv:
+            argv.append('--force')
+
+        if command == 'order':
+            argv = [command, *argv]
+
+        if command in ('dms', 'drink', 'drinks'):
+            dms_result = subprocess.run(['dms', *argv],
+                                        stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+            result_str = dms_result.stdout.decode('utf-8')
+            await self.master.client.send_message(message.rid, result_str)
+
+    def _create_dmsclient_config_if_missing(self, token: str):
```

Comment:
This seems to be a bit much of the dms clients internal. Should be improved /moved to the client but ok for now. I still see one issue though. If the token changes in the botconfig it will not be updated in the dmsrc

Reply:
Yes, should be definitly be handled by a higher-level dmsclient api. Is more a quickfix for now.
That it doesn't overwrite your lokal token is on purpose. Such you don't loose your token if you use dmsclient on your development machine. 
In production you usually use this bot in a docker container, then the config will be created with the up to date token.